include/tty/vga.h

```diff
@@ -0,0 +1,50 @@
+#ifndef _VGA_H
+#define _VGA_H
+
+#include "kernel/kernel.h"
+#include "machine/pt.h"
+
+#define VGA_PHYS_ADDR 0xb8000
+#define VGA_VIRT_BASE phys_to_virt(VGA_PHYS_ADDR)
+
+#define VGA_HEIGHT 80
+#define VGA_WIDTH 20
```

Comment:
i searched about what is "vga" meaning, and on my way to know this, i founded that this VGA_HEIGHT, and width are swapped.


so the standard is:
80 Columns(width=80)
20 Rows(height=20)


```suggestion
#define VGA_WIDTH 80
#define VGA_HEIGHT 20
```

Also i see those resources:
![alt text](image.png)
https://en.wikipedia.org/wiki/VGA_text_mode

Reply:
ها اكمل كمان ولا كدا كفايه؟
دا احنا لسه بنسمي الله!!